vyper/compiler/output.py

```diff
@@ -300,9 +300,13 @@ def _build_opcodes(bytecode: bytes) -> str:
 
     while bytecode_sequence:
         op = bytecode_sequence.popleft()
-        opcode_output.append(opcode_map[op])
+        opcode_output.append(opcode_map.get(op, f"VERBATIM_{hex(op)}"))
         if "PUSH" in opcode_output[-1] and opcode_output[-1] != "PUSH0":
             push_len = int(opcode_map[op][4:])
+            # we can have push_len > len(bytecode_sequence) when there is data
+            # (instead of code) at end of contract
+            # CMC 2023-07-13 maybe just strip known data segments?
```

Comment:
what does this mean?

Reply:
this has to do with an evm peculiarity, which is that you can have truncated push instructions at the end of a contract's bytecode (for instance, the last byte of bytecode is `0x7f`). the comment here is explaining that we deal with that case.

Reply:
oh I actually meant "CMC 2023-07-13 maybe just strip known data segments?"

Reply:
ah - we could strip known data segments so we are not trying to disassemble non-code segments

Reply:
CMC => Mr. King Charles :D; it's resolved